.circleci/config.yml

```diff
@@ -1,21 +1,22 @@
 ---
 common-steps:
-  - &removevirtualenv
+  - &install_deps
     run:
-      name: Removes the upstream virtualenv from the original container image
-      command: sudo pip uninstall virtualenv -y
-
+      name: Install base dependencies for Debian python
+      command: |
+        set -e
+        apt-get update && apt-get install -y sudo make git git-lfs gnupg python3 python3-venv
   - &install_packages
     run:
       name: Install packages
       command: |
-        sudo apt install libnotify-bin
+        sudo apt install -y libnotify-bin
 
   - &run_tests
     run:
       name: Install test requirements and run tests
       command: |
-        virtualenv .venv
+        python3 -m venv .venv
```

Comment:
note: often we just run `make venv` here, which will install the dev requirements